app/models/move.rb

```diff
@@ -116,6 +116,15 @@ class Move < VersionedModel
   before_validation :set_reference
   before_validation :set_move_type
 
+  after_create do
```

Comment:
@Thource  thinking more about this, maybe the model is the wrong place for this code. How about instead moving this to `create_and_render` in `app/controllers/api/v2/moves_actions.rb` AND to `app/controllers/api/v1/moves_actions.rb` Those action files are where do other Nomis actions, e.g. for allocations. 

Keeping the Nomis link outside of the model will simplify the test changes, as only creates via the api will trigger a call to Nomis.

Reply:
That sounds good to me, I'll get on it :)